src/more-infos/more-info-cover.html

```diff
@@ -36,7 +35,7 @@
           on-change='coverPositionSliderChanged'></paper-slider>
       </div>
 
-      <div class='current_tilt_position'>
+      <div class='current_tilt_position' hidden$="[[!supportsTilt]]">
```

Comment:
@andrey-git is right. Have a look at how it is done at [`more-info-light`](https://github.com/home-assistant/home-assistant-polymer/blob/master/src/more-infos/more-info-light.html#L161)